sdnfv/openNetVM
# Launch script

From [Issue 107](https://github.com/sdnfv/openNetVM/issues/107), go script couldn't handle multi-word string arguments


## Summary:
Specifically in the `payload_scan` NF, running with the required `./go.sh 1 -d 1 -s "My String Here"` would not work with multiple words. While `./go.sh 1 -d 1 -s computer` or "computer" works well, bash strips the quotes with variable assignment, so "computer here" would become computer here. Thus the c `getopt` would not recognize the word `here`. 

**Usage:**


| This PR includes         |          |
| ------------------------ | -------- |
| Resolves issues          |  Resolves #107  
| Breaking API changes     |
| Internal API changes     |
| Usability improvements   |  
| Bug fixes                | 👍
| New functionality        |
| New NF/onvm_mgr args     | 
| Changes to starting NFs  |  
| Dependency updates       | 
| Web stats updates        | 



## Merging notes:
 - Dependencies: None  

**TODO before merging :**



## Test Plan:
Run multiple NFs with tons of arguments to ensure the go script doesn't fail. Make sure to run with DPDK arguments as well, to check that errors are handled correctly still. Only adding DPDK without ONVM args should fail, from previous edits to go scripts. 

- Sanity checks, assigned to @koolzz  @dennisafa 
  - Run make in /onvm and /examples directories
  - Code style, assigned to @koolzz  @dennisafa 

- Run linter
  - Check everything style related
  - Code design, assigned to @koolzz  @dennisafa 

- Check for scripting errors
  - Check that code is reusable
  - Code is clean, functions are concise
  - Verify that edge cases properly handled

- Performance, assigned to @koolzz  @dennisafa 
  - Run Speed Tester NF, report performance.
